modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java

```diff
@@ -396,4 +396,6 @@
 
     public static final String USE_ONEOF_DISCRIMINATOR_LOOKUP = "useOneOfDiscriminatorLookup";
     public static final String USE_ONEOF_DISCRIMINATOR_LOOKUP_DESC = "Use the discriminator's mapping in oneOf to speed up the model lookup. IMPORTANT: Validation (e.g. one and only one match in oneOf's schemas) will be skipped.";
+    public static final String INIT_REQUIRED_VARS = "initRequiredVars";
+    public static final String INIT_REQUIRED_VARS_DESC = "If set to false the original beaviour of required vars will be used and if set to true the older beaviour of required vars will be used (behaviour prior to PR #8802)";
```

Comment:
People aren't going to take time to read a PR. Can you fully describe what it does?
something like
When true required variables are included as positional arguments in init and from openapi data. Note: this can break some composition use cases. To learn more read PR blah.

Reply:
Changed the Description for additional property.
https://github.com/OpenAPITools/openapi-generator/pull/10809/files#diff-e1b4dad5dfdcaee3164087471224118ebb7958563501fec4171518883a8887c6R400